target-platform/target-platform.target

```diff
@@ -32,15 +32,6 @@
 			<unit id="org.eclipse.wst.common.emf" version="0.0.0"/>
 			<unit id="org.eclipse.wst.xsd.core" version="0.0.0"/>
 		</location>
-		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/wildwebdeveloper/releases/1.3.4/"/>
-			<repository location="https://download.eclipse.org/tm4e/releases/0.11.0/"/>
-			<repository location="https://download.eclipse.org/lsp4e/releases/0.26.0/"/>
-			<repository location="https://download.eclipse.org/lsp4j/updates/releases/0.22.0/"/>
-			<repository location="https://download.eclipse.org/mylyn/updates/release/4.2.0/"/>
-			<unit id="org.eclipse.wildwebdeveloper.xml.feature.feature.group" version="0.0.0"/>
-			<unit id="org.eclipse.lsp4e" version="0.0.0"/>
-		</location>
```

Comment:
Could you please revert the formatting changes that also remove comments.
The Target Editor unfortunately does not respect the existing element order.

Reply:
I can do that but on the long run we either should not put comments in the target or enhance PDE to retain them in some way otherwise one is always forced to do text-editing targets